packages/jest-util/package.json

```diff
@@ -13,8 +13,7 @@
     "graceful-fs": "^4.1.11",
     "is-ci": "^1.0.10",
     "jest-message-util": "^22.4.0",
-    "mkdirp": "^0.5.1",
-    "source-map": "^0.6.0"
+    "mkdirp": "^0.5.1"
```

Comment:
not sure why this and `yarn.lock` got messed up. am i using an old yarn version on something? or is master currently out of sync?

Reply:
this is weird - source-map was just added in #5594. Maybe you need a rebase?

running `yarn` on master for me does not change the lockfile either